README.md

```diff
@@ -1,125 +1,162 @@
-# lookout [![Build Status](https://travis-ci.org/src-d/lookout.svg)](https://travis-ci.org/src-d/lookout) [![GoDoc](https://godoc.org/gopkg.in/src-d/lookout?status.svg)](https://godoc.org/github.com/src-d/lookout)
+lookout
+[![Build Status](https://travis-ci.org/src-d/lookout.svg)](https://travis-ci.org/src-d/lookout)
+[![GoDoc](https://godoc.org/gopkg.in/src-d/lookout?status.svg)](https://godoc.org/github.com/src-d/lookout)
+[![Code Coverage](https://codecov.io/github/src-d/lookout/coverage.svg)](https://codecov.io/github/src-d/lookout)
+[![Go Report Card](https://goreportcard.com/badge/github.com/src-d/lookout)](https://goreportcard.com/report/github.com/src-d/lookout)
+![development](https://svg-badge.appspot.com/badge/stability/development?color=D6604A)
+=======
 
 A service for assisted code review, that allows running custom code Analyzers on pull requests.
 
-# SDK
+# Configuring lookout
```

Comment:
I suggest moving this section down in ToC, right before `# SDK for Analyzer Developers`

Reply:
But in that case, you go to the easier use case, running with docker compose, and it mentions `config.yml`.

And it's actually a requirement, there is not default for this file. I think it makes sense to have it right in the beginning of the document, so you know there is a reference for this somewhere.

Reply:
Up to you, but the reason for suggestion is - a first time user rarely wants to configure something, he usually needs to understand how to run it first, and then configure

Reply:
I agree with the reasoning. In this case since it's just a link to the config guide, I think it makes sense to keep it so as a first-time user you know that the reference exists, and can come back later.